list.go

```diff
@@ -0,0 +1,85 @@
+// +build linux
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"text/tabwriter"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/codegangsta/cli"
+)
+
+var listCommand = cli.Command{
+	Name:  "list",
+	Usage: "lists containers started by runc with the given root",
+	Action: func(context *cli.Context) {
+
+		// preload the container factory
+		if factory == nil {
+			err := factoryPreload(context)
+			if err != nil {
+				logrus.Fatal(err)
+				return
+			}
+		}
+
+		// get the list of containers
+		root := context.GlobalString("root")
+		absRoot, err := filepath.Abs(root)
+		if err != nil {
+			logrus.Fatal(err)
+			return
+		}
+		list, err := ioutil.ReadDir(absRoot)
+
+		w := tabwriter.NewWriter(os.Stdout, 12, 1, 3, ' ', 0)
+		fmt.Fprint(w, "ID\tSTARTED\tSTATUS\tPID\tROOTFS\n")
```

Comment:
Ok.. Right.. my thoughts regarding pointing them to the bundle directory so they can list the config.json are back with the concern that the config.json might not even be there any more, and certainly could have been edited by then by some higher level tool that generated it and/or is managing it. I was thinking runc list is more for what is going on after runc has loaded the config.json than telling them where the files were loaded from to launch it. At one point we were discussing a need to dump the state data including the copy of the config.json that was used to start the container.. I think it would make sense to have a command at some point that regenerates that which created the container.. this to enable cloning and introspection operations. 

That said.. yeah sure as long as rootfs is allways going to be a subdirectory of the bundle path and never a full path.., I can just tail the rootfs path (using our copy of rootfs in the config.json in memory) to form the bundle path.. and then output the bundle (path) and the relative offset  rootfs.  WDYT?